---
wr8_ai/BoundingBox[] bboxes
